onebusaway-android/build.gradle

```diff
@@ -39,19 +40,30 @@ repositories {
         // CUTR Releases
         url "https://github.com/CUTR-at-USF/cutr-mvn-repo/raw/master/snapshots"
     }
+
+    maven {
+        // for testlib
+        url  "http://dl.bintray.com/acrowntest/test"
+    }
 }
 
 android {
     compileSdkVersion 23
     buildToolsVersion "25.0.2"
 
     defaultConfig {
-        minSdkVersion 9
+        minSdkVersion 15
         targetSdkVersion 21
         versionCode 72
         versionName "2.1.5"
-    }
 
+        // multi dex enabled
```

Comment:
Is the multi dex still required on the release version of the embedded social library?  I'd prefer to avoid multi dex if possible.